### PR TITLE
Ht 7/display list of bookings for next 2 weeks

### PR DESCRIPTION
**Pull request by kalibria**
Created a booking list that contains current date and next 13 dates.

### CODE REVIEW

File: src/booking-list/BookingList.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      </li>
+    );
+  });
+
+  return (
+    <div className={'bookingPage'}>
+      <div className={'bookingWrapper'}>
+        <div className={'bookingHeader cellDecoration'}>Booking list</div>
+        <ul>{datesList}</ul>
+        <ul>booking time</ul>
```
Comment (ikushlianski) [inline src/booking-list/BookingList.tsx:19]: Shouldn't those be table elements?

File: src/booking-list/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const amountOfDays = 14;
```
Comment (ikushlianski) [inline src/booking-list/constants.ts]: constants are usually spelled as `AMOUNT_OF_DAYS`

File: src/booking-list/datesManager.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { amountOfDays } from 'src/booking-list/constants';
+
+class DatesManager {
+  getDatesMS() {
+    let arrOfDays: number[] = [];
+    arrOfDays.length = amountOfDays - 1;
+    arrOfDays.fill(0);
+
+    let currentDate = new Date();
```
Comment (ikushlianski) [inline src/booking-list/datesManager.ts]: try to use `const` for things you don't mutate in subsequent code

File: src/booking-list/BookingList.tsx (hunk 1 above)
Comment (kalibria) [reply src/booking-list/BookingList.tsx:19]: resolved

File: src/booking-list/constants.ts (hunk 2 above)
Comment (kalibria) [reply src/booking-list/constants.ts]: renamed

File: src/booking-list/datesManager.ts (hunk 3 above)
Comment (kalibria) [reply src/booking-list/datesManager.ts]: did

**Review by ikushlianski — APPROVED** [review]

**Pull request merged**